README.md

```diff
@@ -1,2 +1,77 @@
-# javascript-capstone
-javascript-capstone
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# MealzDB Web Application.
+
+- This is a web application that allows you to get information about various kinds of meals and recipes from all over the world.
+- Using, MealzDB you can get information about various meals and recipes from all over the world.
+- Using Interactive API You can make comments live and make reservations for meals. Also you can even like meals and add them to your faviourites. :heart_eyes:
+
```

Comment:
[OPTIONAL] It would be great to add a screenshot of your app, this is a good way to show off your project.